imkar/utils/utils.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import pyfar as pf
+
+
+def paris_formula(coefficients, incident_directions):
+    r"""
+    Calculate the random-incidence coefficient from free-field
+    data for several incident directions.
+
+    Uses the Paris formula [2]_.
+
+    .. math::
+        c_{rand} = \sum c(\vartheta,\varphi) \cdot cos(\vartheta) \cdot
+        w(\vartheta,\varphi)
+
+    with the coefficients :math:`c(\vartheta,\varphi)`, the area
+    weights ``w`` from the `incident_directions.weights`,
+    and :math:`\vartheta` and :math:`\varphi` are the incidence
+    angle and azimuth angles. Note that the incident directions should be
```

Comment:
the term 'incedence angle and azimuth anges' is a bit confusing to me. Could we use only 'incidence angles'?

Reply:
solved it, by directly refering to the pyfar.Coordinates angles